app/components/Modal/WithdrawModalNew.jsx

```diff
@@ -838,6 +840,19 @@ class WithdrawModalNew extends React.Component {
         }
     }
 
+    handleQrScanSuccess(data) {
+        if (!this.state.quantity) {
```

Comment:
Shouldn't this be checking if `data` contains an `amount` field?

Reply:
@svk31 This if checks if user put something on "quantity" field on modal by himself. If yes I do not put amount detected on QR code. If user don't put "quantity" by himself I put amount to quantity. I put a comment for this if.